ChangeLog.md

```diff
@@ -17,6 +17,7 @@
 * Added H3:SOD cheat codes as alternative to vcmi cheats

```

Comment:
For AI 
* Reworked object prioritizing
* Improved town defense against enemy heroes
* Improved town building (mage guild and horde)
* Various behavior fixes